### DISCUSSION TITLE
Cardputer M5Stack

### DISCUSSION THREAD

**Discussion by darklite80** [Wildtimo/M5StickC-Plus-TLite-CFW#1] [Ideas]
have you thought about porting this over to the cardputer and using the thermal 2grove module? i am new to M5 devices and programming. i know it is possible to port stick programs to the cardputer because of the similar hardware but wanted to know if you have any experience doing so.

**Comment by Wildtimo** [comment]
![19A7E5C5-1845-46FC-9F31-CBA64EDC73DE](https://github.com/Wildtimo/M5StickC-Plus-TLite-CFW/assets/114223553/45009265-3d78-409f-a989-bb3c0db71e62)

(Image from t-lite review^) if this isn’t the thermal 2grove you were talking about tell me.

I haven’t thought about porting as I don’t have a cardputer, but it should be relatively easy considering the compatible specs, (same screen, and compatible esp, and based on the image you can use the grove port)

all you need to do is redefine the pins for stuff and get the buttons mapped to the keyboard.

I think it might be possible to allow scrolling up in menus since we have a keyboard. :D

**Reply by darklite80** [reply]
![image](https://github.com/Wildtimo/M5StickC-Plus-TLite-CFW/assets/48372286/4b6e3495-bdd8-4ab7-8e05-2193dfc59058)

**Reply by darklite80** [reply]
![image](https://github.com/Wildtimo/M5StickC-Plus-TLite-CFW/assets/48372286/5e78267e-5bb7-49e9-b3d1-af42a1fab5e4)

**Reply by darklite80** [reply]
Looks to be the same thermal module but the one you showed was the pimaroni one not sure if any differences in the 2
